src/connection.ts

```diff
@@ -36,6 +38,8 @@ export const SFDX_HTTP_HEADERS = {
   'user-agent': clientId,
 };
 
+const ORG_NOT_FOUND_ERROR_NAME = 'Org Not Found';
```

Comment:
Please consider changing this to "Domain Not Found", which reflects the actual error encountered. While a low risk, it is possible that the org's domain name was changed and the local user json file reflects the old domain name in instance Url. [Rename Your My Domain](https://help.salesforce.com/articleView?id=domain_name_rename.htm&type=5)

Reply:
Good point.